src/components/Reservations/Reservation.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { deleteReservation } from '../../redux/reservations/reservations';
+
+const Reservation = (props) => {
+  const dispatch = useDispatch();
+  const {
+    model, city, date, id, avatar,
+  } = props;
+  const handleDelete = () => {
+    dispatch(deleteReservation(id));
+  };
+  return (
+    <div className="card">
+      <div className="card-img">
+        <img src={avatar} alt="avatar" />
+      </div>
+      <div className="card-details">
+        <h4>
+          { model }
+          {' '}
+        </h4>
+        {/* <h4> { city } </h4> */}
```

Comment:
- [ ] Please, remove this old piece of commented code, in order to improve the cleanliness of the repo 💪